keycloak/docker-compose.yml

```
@@ -41,6 +41,24 @@ services:
       - internal
     restart: unless-stopped
 
+  account-ms:
+    image: aamdigital/account-ms:latest
+    depends_on:
+      - keycloak
+    networks:
+      - internal
+      - nginx-proxy_default
+    environment:
+      LETSENCRYPT_HOST: example.com
+      VIRTUAL_HOST: example.com
+      VIRTUAL_PORT: 3000
+      CORS: "*"
+      SENTRY_DSN:
+      KEYCLOAK_URL: http://keycloak:8080
```

```suggestion
      KEYCLOAK_URL: https://example.com
```
This needs to be a https connection or Keycloak has to be configured differently